miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/IlluminaNotificationDto.java

```diff
@@ -10,15 +10,16 @@
 
 public class IlluminaNotificationDto extends NotificationDto {
 
+  private int bclCount;
   private int callCycle;
   private IlluminaChemistry chemistry;
   private int imgCycle;
   private List<Integer> indexLengths;
   private int numCycles;
+  private int numReads;
   private Map<Integer, String> poolNames;
   private int readLength;
   private int scoreCycle;
-  private int bclCount;
 
   @Override
   public boolean equals(Object obj) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 19 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)